registry/registry.go

```diff
@@ -63,6 +57,15 @@ var cipherSuites = map[string]uint16{
 	"TLS_AES_128_GCM_SHA256":       tls.TLS_AES_128_GCM_SHA256,
 	"TLS_AES_256_GCM_SHA384":       tls.TLS_AES_256_GCM_SHA384,
 	"TLS_CHACHA20_POLY1305_SHA256": tls.TLS_CHACHA20_POLY1305_SHA256,
+
+	/* The following cipher suites are insecure, so we disable those options.
+	TLS_RSA_WITH_RC4_128_SHA.
+	TLS_RSA_WITH_AES_128_CBC_SHA256.
+	TLS_ECDHE_ECDSA_WITH_RC4_128_SHA.
+	TLS_ECDHE_RSA_WITH_RC4_128_SHA.
+	TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256.
+	TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256.
+	*/
```

Comment:
It's better to move this commit to a separate pull request, otherwise the security change might get overlooked (burried together with unrelated changes).

I'd also just remove the commented code (as it has no real purpose), and instead putting this information in the commit message, e.g.

```
This commit removes the following cipher suites that are known to be insecure:

    TLS_RSA_WITH_RC4_128_SHA
    TLS_RSA_WITH_AES_128_CBC_SHA256
    TLS_ECDHE_ECDSA_WITH_RC4_128_SHA
    TLS_ECDHE_RSA_WITH_RC4_128_SHA
    TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256
    TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
```

That said, perhaps we should follow the recommendation that's posted in CI (if that works):


> The following example shows how to create a safer TLS configuration:
> 
> ```go
> package main
> 
> import "crypto/tls"
> 
> func saferTLSConfig() {
>     config := &tls.Config{}
>     config.MinVersion = tls.VersionTLS12
>     config.MaxVersion = tls.VersionTLS13
>     // OR
>     config.MaxVersion = 0 // GOOD: Setting MaxVersion to 0 means that the highest version available in the package will be used.
> }
> ```
>

Reply:
Got it!